addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml

```diff
@@ -5,18 +5,11 @@
             <t t-set-slot="layout-actions" t-if="!state.activeDashboard?.isSample">
                 <t t-set="status" t-value="state.activeDashboard and state.activeDashboard.status"/>
                 <div class="d-flex flex-wrap">
-                    <div t-if="status === Status.Loaded"
-                        class="o_filter_value_container"
-                        t-as="filter"
-                        t-foreach="filters"
-                        t-key="activeDashboardId + '_' + filter.id">
-                        <FilterValue
-                            filter="filter"
-                            model="state.activeDashboard.model"
-                            showTitle="true"
-                            showClear="true"
-                        />
-                    </div>
+                    <DashboardSearchBar t-if="status === Status.Loaded"
+                                        t-key="activeDashboardId"
+                                        filters="filters"
```

Comment:
do we need the `filters` props ? `DashboardSearchBar` can get them itself with `props.model.getters.getGlobalFilters()`